public/assets/styles/_sections/Functionlities.less

```diff
@@ -0,0 +1,92 @@
+#Functionlities {
+  padding: 2em 0;
+  #bigPhonePic {
+    display: none;
+  }
+  .functionality {
+    text-align: center;
+    .icon {
+      display: flex;
+      justify-content: center;
+      padding: 1em 0;
+      .iconContainer {
+        background: white;
+        width: 95px;
+        height: 95px;
+        box-shadow: 0 8px 14px rgba(0, 0, 0, 0.16);
+        border-radius: 50px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+    }
+    .title {
+      text-transform: uppercase;
+      font-size: 14px;
+      letter-spacing: 1.87px;
+    }
+    .text {
+      color: @light-gray;
+      font-size: 18px;
+      line-height: 30px;
+    }
+  }
+}
+
+
+@media screen and (max-width: 768px) {
```

Comment:
If you're going to use both min and max breakpoints, make sure to offset them by a pixel. Take a look at the example in the `_variables.less` file. Specifically, check out these 2 variables:

```
@break-sm-up:   544px;
@break-sm-down: 543px;
```

I would create a `@break-md-down: 767px` variable and use it here.